src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,24 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
```

Comment:
```suggestion
    public List<Bulldozer> get() {
```
специфічні продюсери повинні повертати ліст параметризований наслідниками Machine відповідно до умови

Reply:
Зробив.
Але оскільки getAll в MachineServiceImpl повертає List<Machine>, доводиться робити переприсвоєння через цикл.
Питання:
Простіше б одразу повертати List<Machine> зі специфічного продюсера, але коли вже так, можливо є інший спосіб для getAll?
Дякую!